### DISCUSSION TITLE
Submit your site power by Academia Hugo

### DISCUSSION THREAD

**Discussion by shampaakter** [themefisher/academia-hugo#33] [Show and tell]
Thanks for creating your site with Academia  Hugo. Let's share your website to inspire others. You can share your website link here, we will add it to the list.

**Comment by richardcsuwandi** [comment]
Here's my personal website created using Academia Hugo: [richardcsuwandi.github.io](https://richardcsuwandi.github.io) 

Thanks!

**Comment by stremovsky** [comment]
Hi

My websites:
https://privacybunker.io/
https://databunker.org/
